LeetCodeSolutions/EqualRowAndColumnPairs/EqualRowAndColumnPairs.cs

```diff
@@ -0,0 +1,75 @@
+namespace LeetCodeSolutions;
+
+public class EqualRowAndColumnPairsSolution{
+    private class ArrayTracker{
+        private readonly Dictionary<string, int> allArrs;
+        public ArrayTracker(){
```

Comment:
space between different member types, one is a constructor, the other is a private member